Automattic/wp-calypso
# Add a GS limit notice in the Global Styles sidebar

Just like the other Global Styles limit notices, the new notice is shown only for free and personal plans. The implementation is based on the existing noticed by refactoring the logic we used in the pre-save GS notice.



Related to https://github.com/Automattic/dotcom-forge/issues/1233

## Proposed Changes

* Add a new notice in the Global Styles sidebar based on the `ComplementaryArea/core/edit-site` slot.

## Testing Instructions



* Use ETK build from this branch on your Sandbox
* On a free Simple Site, go to site editor and click on the GS button
* Notice that the upgrade nudge is not displayed
* Make a GS change
* Go to Global Styles sidebar and notice that we now display an upgrade nudge on all pages. 

## Pre-merge Checklist
